src/libpgagroal/configuration.c

```diff
@@ -265,6 +267,20 @@ pgagroal_read_configuration(void* shm, char* filename, bool emitWarnings)
                      unknown = true;
                   }
                }
+               else if (key_in_section("metrics_cache_max_age", section, key, true, &unknown))
+               {
+                  if (as_seconds(value, &config->metrics_cache_max_age, PGAGROAL_PROMETHEUS_CACHE_DISABLED))
+                  {
+                     unknown = true;
+                  }
+               }
+               else if (key_in_section("metrics_cache_max_size", section, key, true, &unknown))
+               {
+                  if (as_bytes(value, &config->metrics_cache_max_size, PROMETHEUS_MAX_CACHE_SIZE))
```

Comment:
We need to pass in `PROMETHEUS_DEFAULT_CACHE_SIZE` as well -- capped at `PROMETHEUS_MAX_CACHE_SIZE`

Reply:
Done.